### ISSUE TITLE
_pickle.Pickling Error

### ISSUE THREAD

**Issue by XinyuGuoQAQ** [Decadz/Evolved-Model-Agnostic-Loss#1]
An error bellow occurs when I try to run "run_learning.py". 
```
Traceback (most recent call last):
  File "/home/Evolved-Model-Agnostic-Loss/experiments/run_learning.py", line 383, in <module>
    experiment_executors[args.method](dataset_fn, model_fn, config, random_state)
  File "/home/Evolved-Model-Agnostic-Loss/experiments/run_learning.py", line 280, in _run_evomal_experiment
    export_loss(lfl.meta_loss_function, res_directory, file_name)
  File "/home/Evolved-Model-Agnostic-Loss/experiments/resources/exporter.py", line 23, in export_loss
    torch.save(loss_function, res_directory + "loss_functions/" + file_name + ".pth")
  File "/home/anaconda3/envs/torch112/lib/python3.8/site-packages/torch/serialization.py", line 423, in save
    _save(obj, opened_zipfile, pickle_module, pickle_protocol)
  File "/home/anaconda3/envs/torch112/lib/python3.8/site-packages/torch/serialization.py", line 635, in _save
    pickler.dump(obj)
_pickle.PicklingError: Can't pickle <function <lambda> at 0x7fac22cdd430>: attribute lookup <lambda> on source.representation.utilities.gp_expression failed
```

The error occurred in the line of code saving the meta-learned loss function. 
```
torch.save(loss_function, res_directory + "loss_functions/" + file_name + ".pth")
```
It seems that it doesn't support function which is in the form of lambda. But I have no other solution. 
any suggestions?

**Comment by Decadz** [comment]
Hi @XinyuGuoQAQ ,

I reran `run_learning.py` on my computer and everything seems to be working fine. I have not been able to reproduce your error. Can I have some information about how you are using the library to see if we can isolate the issue? 

**Specifically:**

1. What is the terminal command you used for running `run_learning.py`, which gave the error?
2. Have you made any changes to the code, _e.g._ any changes to the function/terminal set?
3. What DEAP version are you using? I tried on 1.3.1, 1.3.2, and 1.3.3 and it seems to be working.

 **Additionally:** 

Can you try the following terminal command to see if that works for you:

`python experiments/run_learning.py --method evomal --dataset california --model mlp --seeds 0 --num_generations 5 --population_size 5`

Cheers,
Christian

**Comment by XinyuGuoQAQ** [comment]
Hi, Christian
Thanks for your reply.

I previously used the commands you provided in the readme file to run. The package version I am using is deap=1.4.1. I note that the version of deap is different. So I reinstalled version 1.3.1 of deap and I ran the command you provided just now.
The program no longer reported any errors. It seems to be a version issue with the deap package.

Anyway, thank you for your reply. It's very helpful!

Cheers,
Xinyu
